File: CheckMyStar.Bll.Requests/CriterionSaveRequest.cs
using CheckMyStar.Bll.Models;

namespace CheckMyStar.Bll.Requests
{
    public class CriterionSaveRequest
    {
        public StarLevelCriterionModel? StarLevelCriterion { get; set; }
        public StarCriterionModel? StarCriterion { get; set; }
    }
}

File: CheckMyStar.Angular/src/app/www/40_Requests/BackOffice/Criterion-save.request.ts
import { StarCriterionModel } from '../../20_Models/BackOffice/Criterion.model';
import { StarLevelCriterionModel } from '../../20_Models/BackOffice/StarLevelCriterion.model';

export interface CriterionSaveRequest {
  starCriterion: StarCriterionModel;
  starLevelCriterion: StarLevelCriterionModel;
}